modules/con_ipi-installation-workflow.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/ipi/ipi-install-prerequisites.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="installation-workflow_{context}"]
+= Installation workflow
+
+The installation program supports interactive mode, but it is recommended to prepare an `install-config.yaml` file in advance, containing the provisioning details for all of the bare-metal hosts.
+
+The administrator adds the relevant cluster details to the `install-config.yaml` file. The `install-config.yaml` file is shipped to the provisioning host. The administrator generates the manifests and verifies all prerequisites.
+
+The installation program enrolls all nodes in the cluster and starts the bootstrap VM. Metal platform components are started as `systemd` services, which have the following containers:
+
+* Ironic-dnsmasq: DHCP server responsible for handing over the IP addresses to the provisioning interface of various nodes on the provisioning network. Ironic-dnsmasq is only enabled when the `provisioningNetwork` is enabled.
```

Comment:
"The DHCP server". ISG. 

The second sentence is an example where passive voice is optional, because you are focusing more on the object than the subject. You can also say, "Ironic-dnsmasq is only enabled when you deploy an {product-title} cluster with a provisioning network, or "When you deploy a cluster with a provisioning network, the installation program enables Ironic-dnsmasq." These are just a few examples of where knowing the subject of the action leads to stronger and clearer sentences.

Reply:
Updated as suggested.